packages/stryker-babel/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "stryker-babel",
+  "version": "0.1.0",
+  "description": "A plugin for babel projects using Stryker",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "tsc -w",
+    "clean": "rimraf \"+(test|src)/**/*+(.d.ts|.js|.map)\" reports",
+    "prebuild": "npm run clean",
+    "build": "tsc -p .",
+    "postbuild": "tslint -p tsconfig.json",
+    "test": "nyc --reporter=html --report-dir=reports/coverage --check-coverage --lines 85 --functions 90 --branches 60 mocha  \"test/helpers/**/*.js\" \"test/unit/**/*Spec.js\" \"test/integration/**/*Spec.js\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stryker-mutator/stryker"
+  },
+  "engines": {
+    "node": ">=4"
+  },
+  "keywords": [
+    "stryker",
+    "stryker-plugin",
+    "javascript",
+    "stryker-transpiler",
+    "babel"
+  ],
+  "bugs": {
+    "url": "https://github.com/stryker-mutator/stryker/issues"
+  },
+  "author": "Sander Koenders <sanderkoenders@gmail.com",
+  "contributors": [
+    "Nico Jansen <jansennico@gmail.com>",
+    "Simon de Lang <simondelang@gmail.com>",
+    "Niek te Grootenhuis <ntegro@hotmail.com>",
+    "Thomas Peters <thomaspeters4@outlook.com>"
+  ],
+  "homepage": "https://github.com/stryker-mutator/stryker/tree/master/packages/stryker-babel#readme",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "babel-core": "^6.26.0",
+    "log4js": "^1.1.1"
+  },
+  "devDependencies": {
+    "@types/babel-core": "^6.25.3",
+    "babel-plugin-transform-es2015-spread": "^6.22.0",
+    "babel-preset-es2015": "^6.24.1",
+    "stryker-api": "^0.11.0"
+  },
+  "peerDependencies": {
+    "stryker-api": "^0.11.0"
```

Comment:
peer dependency to babel-core is missing? Or by design? If it is by design, we should remove the remark in the README.md

Reply:
Fixed! users may not have babel-core directly if they use the babel-cli. But that does have babel-core as a dependency